tools/generate_og_images.py

```
@@ -0,0 +1,58 @@
+    elif content_type == "music":
+        background_color = (0, 0, 0)
+        text_color = (255, 255, 255)
+        font_size = 100
+    elif content_type == "links":
+        background_color = (255, 255, 255)
+        text_color = (0, 0, 0)
+        font_size = 100
+    elif content_type == "books":
+        background_color = (255, 255, 255)
+        text_color = (0, 0, 0)
+        font_size = 100
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)